ui/src/app/pages/admin/admin-institution-edit.tsx

```diff
@@ -756,6 +770,17 @@ export const AdminInstitutionEdit = fp.flow(
         }
       };
 
+      validate.validators.requestAccessUrl = (url: string) => {
+        if (url && url.length > 2000) {
```

Comment:
Thats a good point, i wonder we should add validation in InstitutionController to check the length of request_access_url, in case some one is trying to access the end point via curl rather than UI